libsolidity/interface/CompilerStack.cpp

```diff
@@ -369,6 +372,13 @@ bool CompilerStack::parse()
 			for (auto const& import: ASTNode::filteredNodes<ImportDirective>(source.ast->nodes()))
 			{
 				solAssert(!import->path().empty(), "Import path cannot be empty.");
+				auto it = sources.find(import->path());
```

Comment:
Should probably put a comment here to indicate that this is for stdlib imports. Would also make sense to use full scoping for `sources`, i.e. `stdlib::sources` to make it even more obvious.